src/components/Card.js

```diff
@@ -32,7 +36,9 @@ export default class Card {
     this._cardImgElement = this._element.querySelector('.card__img');
     this._btnLike = this._element.querySelector('.card__btn-like');
     this._btnDelete = this._element.querySelector('.card__btn-delete');
-
+    if (this._isLiked) {
```

Comment:
Needs correction: the code for rendering likes gets duplicated

The code for rendering likes should look like this:

```js
// in Card class 

// call this method from index.js after successful api response in `then` block
setIsLiked(isLiked) { 
   // set instance variable
    this._isLiked = isLiked; 
    this._renderLikes(); 
} 

isLiked() {
  return this._isLiked;
}

_renderLikes() {
  if (this._isLiked) {
    // add active class to the like button
  } else {
    // remove active class from the like button
  }
}

createCard() { 
  ...
  // render initial likes, you saved them in the constructor
  this._renderLikes()
}

// index.js
if (card.isLiked()) {
  api.dislikeCard(...).then(response => card.setIsLiked(response._isLiked)).catch(() => /*handle error*/)
} else {
 // the same for adding like
}
```